Adds account owned tokens getting started (#17105)
cloudflare/cloudflare-docs: src/content/partials/fundamentals/account-owned-tokens.mdx

---

[]

---

## What are Account Owned Tokens?

> Account Owned Tokens are a first step that Cloudflare is taking in order to represent service principals in our service, and they are being released currently as an Open Beta.

Reviewer: ```suggestion
Account Owned Tokens are Cloudflare's first step to represent service principals in our service. Account Owned Tokens are currently in Open Beta.
```

What is "in our service" referring to?

Author: The Cloudflare system.

> Engineers across the company are working to ensure that all features eventually become compatible with Account Owned Tokens, and we have provided a short compatability list below.

Reviewer: ```suggestion
Cloudflare is working to enable all features to be compatible with Account Owned tokens, and a short compatibility list is provided below.
```

> If you are working with a service that is not currently supported by account owned tokens please continue to use the existing user tokens.

Reviewer: ```suggestion
If you are working with a product or feature that is not currently supported by Account Owned tokens, continue to use the existing user tokens.
```

User tokens will continue to work, and we do not have plans to deprecate them.

> Account Owned Tokens will be visible for Super Administrators of accounts on the Cloudflare dashboard via **Manage Account** > **API Tokens**.

Reviewer: ```suggestion
Account Owned Tokens are visible for Super Administrators of accounts on the Cloudflare dashboard via **Manage Account** > **API Tokens**.
```

> The existing token creation UI is available, and can also be accessed from the API at /accounts/:accountID/tokens

Reviewer: ```suggestion
The existing token creation UI is available and can also be accessed from the API at /accounts/:accountID/tokens.
```

Try using Account Owned Tokens specifically in these scenarios:

> 1. You require business continuity when managing tokens as a team of super administrators

Reviewer: ```suggestion
1. If you require business continuity when managing tokens as a team of super administrators
```

> 2. You need to restrict API access on your account, and want to centralize visibility and management of these tokens

Reviewer: ```suggestion
2. If you need to restrict API access on your account and want to centralize visibility and management of these tokens
```

## Account Owned Tokens Compatibility Matrix

> Account Owned Tokens are a new credential type that is in Open Beta. This is the list of currently supported products and their compatability status.

Reviewer: Just noting that lists like this tend to get out of date quickly - may be a good idea to consider (after birthday week!) figuring out how to dynamically update

## What are Account Owned Tokens?

Account Owned Tokens are Cloudflare's first step to represent service principals in our service. Account Owned Tokens are currently in Open Beta.
Cloudflare is working to enable all features to be compatible with Account Owned tokens, and a short compatibility list is provided below.

> If you are working with a product or feature that is not currently supported by Account Owned tokens, continue to use the existing user tokens.

Reviewer: ```suggestion
If you are working with a product or feature that is not currently supported by Account Owned Tokens, continue to use the existing user tokens.
```
